sdk/cdn/arm-cdn/package.json

```diff
@@ -29,22 +29,23 @@
   "types": "./types/arm-cdn.d.ts",
   "devDependencies": {
     "@microsoft/api-extractor": "^7.31.1",
-    "mkdirp": "^1.0.4",
+    "mkdirp": "^2.1.2",
```

Comment:
https://github.com/Azure/autorest.typescript/pull/2436/files can you check if other places for this repo has changed mkdirp to v3 ?